src/timeseriesinsights/README.md

```diff
@@ -1,95 +1,122 @@
-Microsoft Azure CLI 'timeseriesinsights' Extension
-==========================================
+# Azure CLI timeseriesinsights Extension
+This is the extension for timeseriesinsights
 
-This package is for the 'timeseriesinsights' extension:
-
-```sh
-az timeseriesinsights
+### How to use
+Install this extension using the below CLI command
 ```
-
-## Install
-
-```sh
 az extension add --name timeseriesinsights
 ```
 
-## Uninstall
-
-```sh
-az extension remove --name timeseriesinsights
+### Included Features
+#### timeseriesinsights environment
+##### Gen1 Create
 ```
-
-## Examples
-
-All commands are for Bash. For PowerShell, please set the variables like 
-
-```powershell
-$rg='rg1'
+az tsi environment gen1 create --name "env1" --location westus --data-retention-time "P31D" --partition-key-properties name="DeviceId1" type="String" --sku name="S1" capacity=1 --resource-group "rg1"
 ```
 
-### Create a resource group for the environments
-
-```sh
-rg={resource_group_name}
-az group create --name $rg --location westus
+##### Gen1 Update
 ```
-
-### Create a standard environment
-
-```sh
-env={standard_environment_name}
-az timeseriesinsights environment standard create -g $rg --name $env --location westus --sku-name S1 --sku-capacity 1 --data-retention-time 31 --partition-key-properties DeviceId1 --storage-limit-exceeded-behavior PauseIngress
+az tsi environment gen1 update --name "env1" --sku name="S1" capacity=2 --resource-group "rg1" --data-retention-time "P30D" --storage-limit-exceeded-behavior PurgeOldData
 ```
 
-### Create a storage account and use it to create a long-term environment
-
-```sh
-storage={storage_account_name}
-env_lt={longterm_environment_name}
+##### Gen2 Create
+```
+az tsi environment gen2 create --name "env2" --location westus --resource-group "rg1" --sku name="L1" capacity=1 --time-series-id-properties name=idName type=String --storage-configuration account-name=your-account-name  management-key=your-account-key
+```
 
-az storage account create -g $rg -n $storage --https-only
-key=$(az storage account keys list -g $rg -n $storage --query [0].value --output tsv)
+##### Gen2 Update
+```
+az tsi environment gen2 update --name "env2" --resource-group "rg1" --warm-store-configuration data-retention=P30D --storage-configuration account-name=your-account-name management-key=your-account-key
+```
 
-az timeseriesinsights environment longterm create -g $rg --name $env_lt --location westus --sku-name L1 --sku-capacity 1 --data-retention 7 --time-series-id-properties DeviceId1 --storage-account-name $storage --storage-management-key $key
+##### Show
+```
+az tsi environment show --name "env1" --resource-group "rg1"
 ```
 
-### Create an event hub and use it to create an event source
+##### List
+```
+az tsi environment list --resource-group "rg1"
+```
 
+##### Delete
+```
+az tsi environment delete --name "env1" --resource-group "rg1"
+```
+#### timeseriesinsights event-source
+##### Event Hub Create
```

Comment:
I think the changed title can't reflect the relationship between the Event Hub and TSI event source.